src/main/java/com/adobe/aem/compgenerator/utils/DialogUtils.java

```diff
@@ -163,9 +163,10 @@ private static Element createPropertyNode(Document document, Node currentNode, P
         // Some of the properties are optional based on the different types available.
         addBasicProperties(propertyNode, property);
 
-        if (StringUtils.isNotEmpty(property.getField()) && (!property.getType().equalsIgnoreCase("radiogroup"))
```

Comment:
no idea why "radiogroup" was attempted to be ignored here but that logic wasnt working due to the `||`.  Radiogroup needs these fields, so I simply omitted this check and validated that the output for radiogroup and image fields is the same both before and after the change.